docs/docs/supply-chain/sbom.md

```diff
@@ -453,7 +453,7 @@ SPDXVersion: SPDX-2.2
 DataLicense: CC0-1.0
 SPDXID: SPDXRef-DOCUMENT
 DocumentName: alpine:3.15
-DocumentNamespace: https://aquasecurity.github.io/trivy/container_image/alpine:3.15-bebf6b19-a94c-4e2c-af44-065f63923f48
+DocumentNamespace: https://trivy.dev/container_image/alpine:3.15-bebf6b19-a94c-4e2c-af44-065f63923f48
```

Comment:
nit:
This is the old SPDX format (Trivy currently uses 2.3).
So the diff for this example will be the old format (for the old version of Trivy) and the new link.

I suggest changing the whole example (maybe in another PR).

Reply:
thanks, updated